docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::warning
+This implementation is experimental and subject to change.
+:::
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](./split-keyboards.md) with the dongle as "central". There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- The "central" dongle results in increased battery life for the keyboard/former "central", as it is now a peripheral.
+- It is easier to connect to the host device.
+
+Disadvantages:
+
+- An extra microcontroller is needed.
+- The keyboard becomes unusable without the dongle.
+
+
```

Comment:
added it as a possible disadvantage

Reply:
To clarify, I don't think it should be listed under advantages or disadvantages, but as a note separate from both. This is because whether it is an advantage or a disadvantage depends on how the dongle is used.

Reply:
moved it to a note and kept et vague. ref to split kb section